CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   #4698)
 - Add documentation comments to the output of the documentation generator.
   (@vanillajonathan, #4729)
+- Add CLI syntax highlighting to `prqlc`. (@vanillajonathan, #4755)
```

Comment:
Can we give the command so people can try it out?

Reply:
Okay, I made an edit to clarify it. Please feel free to edit if you can come up with a better changelog description.